src/tables.ts

```diff
@@ -1049,254 +1093,214 @@ export class Tables {
       case 'T2':
-        code = 'noteheadTriangleUpBlack';
-        break;
+        return '\ue0be' /*noteheadTriangleUpBlack*/;
       case 'T3':
-        code = 'noteheadTriangleUpBlack';
-        break;
+        return '\ue0be' /*noteheadTriangleUpBlack*/;
 
       /* Cross */
       case 'X0':
-        code = 'noteheadXWhole';
-        break;
+        return '\ue0a7' /*noteheadXWhole*/;
       case 'X1':
-        code = 'noteheadXHalf';
-        break;
+        return '\ue0a8' /*noteheadXHalf*/;
       case 'X2':
-        code = 'noteheadXBlack';
-        break;
+        return '\ue0a9' /*noteheadXBlack*/;
       case 'X3':
-        code = 'noteheadCircleX';
-        break;
+        return '\ue0b3' /*noteheadCircleX*/;
 
       /* Square */
       case 'S1':
-        code = 'noteheadSquareWhite';
-        break;
+        return '\ue0b8' /*noteheadSquareWhite*/;
       case 'S2':
-        code = 'noteheadSquareBlack';
-        break;
+        return '\ue0b9' /*noteheadSquareBlack*/;
 
       /* Rectangle */
       case 'R1':
-        code = 'vexNoteHeadRectWhite'; // no smufl code
-        break;
+        return '\ue0b8' /*noteheadSquareWhite*/; // no smufl code
       case 'R2':
-        code = 'vexNoteHeadRectBlack'; // no smufl code
-        break;
+        return '\ue0b8' /*noteheadSquareWhite*/; // no smufl code
 
       case 'DO':
-        code = 'noteheadTriangleUpBlack';
-        break;
+        return '\ue0be' /*noteheadTriangleUpBlack*/;
       case 'RE':
-        code = 'noteheadMoonBlack';
-        break;
+        return '\ue0cb' /*noteheadMoonBlack*/;
       case 'MI':
-        code = 'noteheadDiamondBlack';
-        break;
+        return '\ue0db' /*noteheadDiamondBlack*/;
       case 'FA':
-        code = 'noteheadTriangleLeftBlack';
-        break;
+        return '\ue0c0' /*noteheadTriangleLeftBlack*/;
       case 'FAUP':
-        code = 'noteheadTriangleRightBlack';
-        break;
+        return '\ue0c2' /*noteheadTriangleRightBlack*/;
       case 'SO':
-        code = 'noteheadBlack';
-        break;
+        return '\ue0a4' /*noteheadBlack*/;
       case 'LA':
-        code = 'noteheadSquareBlack';
-        break;
+        return '\ue0b9' /*noteheadSquareBlack*/;
       case 'TI':
-        code = 'noteheadTriangleRoundDownBlack';
-        break;
+        return '\ue0cd' /*noteheadTriangleRoundDownBlack*/;
 
-      case 'D':
-      case 'H': // left for backwards compatibility
-        switch (duration) {
-          case '1/2':
-            code = 'noteheadDiamondDoubleWhole';
-            break;
-          case '1':
-            code = 'noteheadDiamondWhole';
-            break;
-          case '2':
-            code = 'noteheadDiamondHalf';
-            break;
-          default:
-            code = 'noteheadDiamondBlack';
-            break;
-        }
-        break;
-      case 'N':
-      case 'G':
+      /* Diamond */
+      case 'DI':
+      case 'H': // Harmonics
```

Comment:
Inconsistent comment style here. Should we just write `// Diamond`?

Reply:
Done